bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -62,4 +62,40 @@
 		</config-description>
 	</bridge-type>
 
+	<!-- API v2 Bridge -->
+	<bridge-type id="bridge-api2">
+		<label>Hue API v2 Bridge</label>
+		<description>The Hue Bridge represents a Philips Hue Bridge supporting API v2.</description>
+
+		<properties>
+			<property name="vendor">Signify</property>
```

Comment:
Probably we need to ask @jlaur about this; he is the one who generally asks for changes Philips -> Signify ;)

But .. IMHO .. even though Signify is the legal name of the company, I think in reality most people still call it the Philips Hue system.

And .. BTW .. the vendor property is in the XML file so it can be applied to non instantiated things (e.g. in the Inbox) .. but when the thing is actually instantiated, the actual properties are downloaded at run time, as shown below..

![image](https://user-images.githubusercontent.com/893994/235657931-ac8263ed-30c3-4a27-9470-b7011d53a522.png)

Reply:
In my case, vendor is "Philips" but I am not sure that it is set at run time (for the API v1).
If set at run time (for API v2), I don't thing there is a need to provide it in the XML description.

Reply:
Signify is the company behind Philips hue nowadays, Philips sold it and doesn't have anything to do with it anymore.

Reply:
@jlaur so what do you want us (myself and/or @lolodomo) to do?

Reply:
> I don't thing there is a need to provide it in the XML description.

I will delete it from v2. Done.

Reply:
Look at manifacturer here:
https://github.com/openhab/openhab-addons/pull/14871#issuecomment-1529615244